build/ansible/pmm2/post-build-actions.yml

```diff
@@ -87,6 +87,45 @@
       when: ansible_virtualization_type == "docker"
       shell: supervisorctl shutdown
 
+      # PMM-11336 - The previous steps failed to start PostgreSQL using supervisord,
+      # so a temporary solution was to start it without supervisord and remove the
+      # pmm-managed database/role. However, a complete overhaul of the pipeline is
+      # necessary for a permanent fix.
+    - name: Start PostgreSQL database without supervisord
+      command: /usr/pgsql-14/bin/pg_ctl start -D /srv/postgres14
+      become: yes
+      become_user: postgres
+
+    - name: Check pmm-managed database existence
+      postgresql_db:
+        login_user: postgres
+        name: pmm-managed
+        state: present
+      register: db_exists
+
+    - name: Remove pmm-managed database
+      postgresql_db:
+        login_user: postgres
+        name: pmm-managed
+        state: absent
+      when: db_exists.db | default(false)
```

Comment:
`when: db_exists.db` - would that suffice?

Reply:
Frankly, I would assume that the role exists after a successful build and:
1/ remove the same tasks from above, i.e. [lines](https://github.com/percona/pmm/pull/1673/files#diff-ca10aefe274646dba1a8c6cf00ce819d94ba0f6e40b3050eeffd9d54643f9046L70-L80)
2/ remove all when statements in db and role deletion tasks

Reply:
That would be much simpler to read and understand.

Reply:
I've committed a simplified version.